### DISCUSSION TITLE
How do you use extenral libs with lit?

### DISCUSSION THREAD

**Discussion by dbwodlf3** [lit/lit#3190] [General]
actually, i just insert document.element, if window.externalLib don't exist.
how do you do? there are better ways?

```ts
// example

//...
  firstUpdated(){
    if(!(window as any).daum){
      const script = document.createElement('script');
      script.src = '//t1.daumcdn.net/mapjsapi/bundle/postcode/prod/postcode.v2.js';
      document.body.appendChild(script);
    } 
  }
//...
```

**Comment by sorvell** [comment]
If the script is available as a module, you can just use [dynamic import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import).
